editor/utils/tracking.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
I don't think this belongs in a new root folder.  Maybe as part of `utils`, but the prompt component should stay in `editor`.